pkg/network/node_identity.go

```diff
@@ -0,0 +1,206 @@
+	return false, nil
+}
+
+// renderNetworkNodeIdentity renders the network node identity component
+func renderNetworkNodeIdentity(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.BootstrapResult, manifestDir string, client cnoclient.Client) ([]*uns.Unstructured, error) {
+	if !bootstrapResult.Infra.NetworkNodeIdentityEnabled {
+		klog.Infof("Network node identity is disabled")
+		return nil, nil
+	}
+	data := render.MakeRenderData()
+	data.Data["ReleaseVersion"] = os.Getenv("RELEASE_VERSION")
+	data.Data["OVNHybridOverlayEnable"] = false
+	if conf.DefaultNetwork.OVNKubernetesConfig != nil {
+		data.Data["OVNHybridOverlayEnable"] = conf.DefaultNetwork.OVNKubernetesConfig.HybridOverlayConfig != nil
+	}
+	data.Data["NetworkNodeIdentityPort"] = NetworkNodeIdentityWebhookPort
+
+	manifestDirs := make([]string, 0, 2)
+	manifestDirs = append(manifestDirs, filepath.Join(manifestDir, "network/node-identity/common"))
+
+	clusterBootstrapFinished := true
+	webhookCAConfigMap := &corev1.ConfigMap{}
+	webhookCAClient := client.Default()
+	webhookCALookup := types.NamespacedName{Name: "network-node-identity-ca", Namespace: NetworkNodeIdentityNamespace}
+	caKey := "ca-bundle.crt"
+
+	webhookReady := false
+	// HyperShift specific
+	if hcpCfg := platform.NewHyperShiftConfig(); hcpCfg.Enabled {
+		webhookCAClient = client.ClientFor(names.ManagementClusterName)
+		webhookCALookup = types.NamespacedName{Name: "openshift-service-ca.crt", Namespace: hcpCfg.Namespace}
+		caKey = "service-ca.crt"
+
+		data.Data["HostedClusterNamespace"] = hcpCfg.Namespace
+		data.Data["ManagementClusterName"] = names.ManagementClusterName
+		data.Data["NetworkNodeIdentityReplicas"] = 1
+		if bootstrapResult.Infra.HostedControlPlane.Spec.ControllerAvailabilityPolicy == hyperv1.HighlyAvailable {
+			data.Data["NetworkNodeIdentityReplicas"] = 3
+		}
+		data.Data["ReleaseImage"] = hcpCfg.ReleaseImage
+		data.Data["CLIImage"] = os.Getenv("CLI_IMAGE")
+		data.Data["TokenMinterImage"] = os.Getenv("TOKEN_MINTER_IMAGE")
+		data.Data["TokenAudience"] = os.Getenv("TOKEN_AUDIENCE")
+		data.Data["HCPNodeSelector"] = bootstrapResult.Infra.HostedControlPlane.Spec.NodeSelector
+		localAPIServer := bootstrapResult.Infra.APIServers[bootstrap.APIServerDefaultLocal]
+		data.Data["K8S_LOCAL_APISERVER"] = "https://" + net.JoinHostPort(localAPIServer.Host, localAPIServer.Port)
+
+		webhookDeployment := &appsv1.Deployment{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "Deployment",
+				APIVersion: appsv1.SchemeGroupVersion.String(),
+			},
+		}
+		nsn := types.NamespacedName{Namespace: hcpCfg.Namespace, Name: "network-node-identity"}
+		if err := client.Default().CRClient().Get(context.TODO(), nsn, webhookDeployment); err != nil {
+			if !apierrors.IsNotFound(err) {
+				return nil, fmt.Errorf("failed to retrieve existing network-node-identity deployment: %w", err)
+			} else {
+				klog.Infof("network-node-identity deployment does not exist")
+			}
+		}
```

Comment:
What happened to https://github.com/openshift/cluster-network-operator/blob/release-4.14/pkg/network/node_identity.go#L114-L116?

Reply:
the function `deploymentProgressing()` does not exist